src/packages/@ncigdc/modern_components/Pvalue/Survival.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```

Comment:
This file isn't used anywhere?

Reply:
oh yea copy/pasted a directory to get started